.github/workflows/update-docker-requirements.yml

```diff
@@ -42,10 +43,22 @@ jobs:
           NEEDS_UPDATE=$(git diff --quiet && echo 'false' || echo 'true')
           echo "needs-update=${NEEDS_UPDATE}" | tee --append $GITHUB_OUTPUT
 
+      - name: Get last merged PR info
+        id: pr_info
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
```

Comment:
Do we really need that? Everything we do here should be public information.

Reply:
I guess yes, as when i tried removing that, it throws this error - [https://github.com/arjxn-py/ragna/actions/runs/10141546877/job/28039000468#step:6:26](https://github.com/arjxn-py/ragna/actions/runs/10141546877/job/28039000468#step:6:26)